rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/config/ArchiveIdentificationConfigLoadingRuleProvider.java

```diff
@@ -36,9 +39,15 @@
         return Initialization.class;
     }
 
+    // @Inject private StaticArchiveIdentifier identifier;
```

Comment:
I don't believe that I understand the purpose of CDI for this particular instance?